pymc3/tests/test_parallel_sampling.py

```diff
@@ -176,30 +192,36 @@ def func(x):
         pm.sample(draws=10, tune=10, step=pm.Metropolis(), cores=2, mp_ctx="spawn")
 
 
+@pytest.mark.xfail(reason="DensityDist was not yet refactored for v4")
 def test_spawn_densitydist_bound_method():
     with pm.Model() as model:
         mu = pm.Normal("mu", 0, 1)
         normal_dist = pm.Normal.dist(mu, 1)
-        obs = pm.DensityDist("density_dist", normal_dist.logp, observed=np.random.randn(100))
+        logp = lambda x: pm.logp(normal_dist, x, transformed=False)
```

Comment:
`pm.logp` doesn't exist. did you mean `logpt`? I would leave these DensityDist unchanged, and just add the xfail mark